File: sem2_c_cpp_java/10-02-23/q1.java
//1. Write a program to read 3 integer numbers that are input by user and print the same onto screen.
import java.util.*;

class Q1{
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        System.out.println("Enter three integers:");
        int n1= input.nextInt();
        int n2= input.nextInt();
        int n3= input.nextInt();
        System.out.println("The Numbers are: " + n1+ " "+ n2+" "+n3);
    }
}

File: sem1_cs/AnmolSingh_225HSBB004/Assignment3_21_09_22/q1.c
#include<stdio.h>

int main()
{
	int arr[10],i;
	
	printf("Enter 8 integers\n");
	for(i=0;i<8;i++)
	{	
		scanf("%d",&arr[i]);
	}
	for(i=0;i<8;i++)
	{
	printf("%d\n",arr[i]);
	}
	return(0);	
}
